packages/build/src/plugins_core/edge_functions/index.js

```diff
@@ -11,7 +11,7 @@ import { parseManifest } from './lib/internal_manifest.js'
 const DENO_CLI_CACHE_DIRECTORY = '.netlify/plugins/deno-cli'
 const IMPORT_MAP_FILENAME = 'edge-functions-import-map.json'
 
-const coreStep = async function ({
+export const coreStep = async function ({
```

Comment:
I don't think you need to export this, since it's already part of the `bundleEdgeFunctions` export (in the `coreStep` property). This is how the core step is executed as part of the main flow.